e2e-tests/UnsignedUsersRedirects.spec.ts

```diff
@@ -7,6 +7,5 @@ test("Homepage redirects to /javascript for unsigned users", async ({ page }) =>
 
 test("Insights Hub redirects to /javascript for unsigned users", async ({ page }) => {
   await page.goto("/hub/insights");
-  await expect(page).toHaveURL("/");
-  await expect(page).toHaveURL(/.*\/javascript.*/, { timeout: 10000 });
+  await expect(page).toHaveURL("http://localhost:3000/javascript/dashboard/filter/recent?redirectedFrom=%2Fhub%2Finsights", { timeout: 10000 });
```

Comment:
I don't think this should include `localhost`. How about just `/javascript/dashboard/filter/recent?redirectedFrom=%2Fhub%2Finsights`

Reply:
That's right :+1: